langlib/lang.array/src/main/ballerina/array.bal

```diff
@@ -169,6 +214,11 @@ public isolated function every(Type[] arr, @isolatedParam function(Type val) ret
 # This removes the member of parameter `arr` with index parameter `index` and returns it.
 # It panics if there is no such member.
 #
+# ```
+# int[] evenNumbers = [2, 4, 6, 8];
+# evenNumbers.remove(1) => 4
```

Comment:
Can't do `[2, 4, 6, 8].remove(1) => 4` since a tuple type is inferred for the list and the removal fails at runtime. Same for some other functions too.

Related to https://github.com/ballerina-platform/ballerina-spec/issues/1050

Reply:
Options,

1. break this into two. Variable definition and expression.
2. Use a type cast with an array literal. 
3. Use let expression

Reply:
I prefer option #2 and I feel we should do that for all the examples not only above.